ghost/core/core/server/data/schema/schema.js

```diff
@@ -824,6 +824,24 @@ module.exports = {
             ['email_id', 'member_email']
         ]
     },
+    email_recipient_failures: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        email_id: {type: 'string', maxlength: 24, nullable: false, references: 'emails.id'},
+        member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id'},
```

Comment:
I'm wondering whether we should delete email_recipient_failures if a member is deleted or not 🤔 We don't delete the email_recipient because it doesn't have a foreign key

Reply:
🤔 Hmm good point, tho it might be weird to have `email_recipient_failures` to be deleted but not `email_recipients` on deleting a member

Reply:
As discussed, have made the `member_id` to be nullable so we can set it to `null` if a member is deleted, and also removed the reference for `member_id`